server/graphql/v1/mutations/collectives.js

```diff
@@ -421,16 +418,6 @@ export function editCollective(_, args, req) {
         // we omit those attributes that have already been updated above
         return collective.update(omit(newCollectiveData, ['HostCollectiveId', 'hostFeePercent', 'currency']));
       })
-      .then(() => collective.editTiers(args.collective.tiers))
-      .then(() => {
-        // @deprecated since 2019-10-21: now using dedicated `editCoreContributors` endpoint
-        if (args.collective.members) {
-          return collective.editMembers(args.collective.members, {
-            CreatedByUserId: req.remoteUser.id,
-            remoteUserCollectiveId: req.remoteUser.CollectiveId,
-          });
-        }
-      })
```

Comment:
I couldn't find a single place in the frontend where we're still using `editCollective` to edit members, even the legacy places like the onboarding modal still use the `editCoreContributors` mutation.